api/src/lib/interface.ts

```diff
@@ -83,6 +83,7 @@ export interface CreatePublicationRequestBody {
     dataAccessStatement?: string;
     selfDeclaration?: boolean;
     affiliationStatement?: string;
+    references?: Reference[];
```

Comment:
I think we can remove this since we have separate endpoints for the references. We update the references for a particular publication separately